openapi/openapi_V1.yaml

```diff
@@ -31,7 +31,9 @@ paths:
                 type: object
                 properties:
                   isPaid:
-                    type: boolean
+                    type: string
```

Comment:
why string?

Reply:
the endpoint: "/check/{rptId}" was removed because it is unused